### DISCUSSION TITLE
Deduplication issues

### DISCUSSION THREAD

**Discussion by j0sien** [asreview/asreview#1292] [Q&A]
For my systematic literature review, I started with a dataset of 19773 records
I found that different deduplication processes lead to different results.

deduplication in R directly only removed a few, result: 19458 left.

So I chose to start with deduplication using asreview datatools, 17224 records were left
after running another deduplication using the R scripts, 16981 records were left
when doing another deduplication round using [SR-accelerator's Deduplicator](https://sr-accelerator.com/#/deduplicator), I'm left with either 11632 records (using the cautious mode) or 10153 (using the 'thourough' mode).

I chose to use the results of the 'cautious' mode, just to be sure not to miss anything.
However, when entering my prior knowledge on 116 records (based on a preliminary screening of 1%), I found 9 records with duplicates (same/similar title and abstract).

**Is there any problem when having some duplicates in the dataset when starting the review process in asreview?**

**If not, is there any problem with identifying multiple records as 'irrelevant', would those influence the model more? or should I only label 1 of those duplicates as '(ir)relevant for prior knowledge and leave the duplicate for what it is?** @daob

**Comment by FelixWdm** [comment]
Hi @j0sien , first of all, duplicates are inevitable with these numbers, and you've already very cautiously excluded a lot of them. Whether excluding the duplicates twice will influence the remaining part of the screening is hard to tell. This will differ per field. If you have the option to remove the duplicates from the prior knowledge that is preferred, but if not, I expect there will not be serious downsides. I hope this helps and let us know if you have more questions.

**Comment by Kirsten-Ziesemer** [comment]
Hi @j0sien you might want to keep an eye out for our research comparing deduplication methods (including R-methods). We just had Dutch Dedup-Day comparing 5 deduplication methods in small groups against a small and medium size benchmarkset (275 and 6602 references). Contact me if you would like to know more!

**Reply by j0sien** [reply]
Hi @Kirsten-Ziesemer yes please keep me informed I am interested in this topic!

**Reply by SHogenboom** [reply]
@Kirsten-Ziesemer ; do you perhaps have a reference for the mentioned de-duplication research? Sounds very interesting and useful! Thanks in advance :)

**Reply by Kirsten-Ziesemer** [reply]
We just presented our preliminary results during #ESMARConf online: https://www.youtube.com/live/Sas4XNjlXgg?feature=share
